aresponses/main.py

```diff
@@ -92,19 +92,17 @@ def _host_matches(self, match_host):
 
         TCPConnector._resolve_host = _resolver_mock
 
-        self._old_update_host = ClientRequest.update_host
+        self._old_is_ssl = ClientRequest.is_ssl
 
-        def new_update_host(_self, *args, **kwargs):
-            val = self._old_update_host(_self, *args, **kwargs)
-            _self._ssl = False
-            return val
+        def new_is_ssl(_self):
+            return False
 
-        ClientRequest.update_host = new_update_host
+        ClientRequest.is_ssl = new_is_ssl
```

Comment:
i am sure you have gone through this thoroughly. i am a little confused because the old way was changing the the `_ssl` attribute which could be passed into the constructor and answered the `ssl` property, but the new way only seems to be passed into the `ConnectionKey`. that is the intention?

Reply:
We discussed this but just documenting.  To be honest I'm not sure exactly what the implications are of doing it this way.  I know the old way no longer worked with aiohttp v3 and that this way did.